satellite/satellitedb/coinpaymentstxs.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellitedb
+
+import (
+	"context"
+	"math/big"
+
+	"storj.io/storj/satellite/payments/coinpayments"
+	"storj.io/storj/satellite/payments/stripecoinpayments"
+	dbx "storj.io/storj/satellite/satellitedb/dbx"
+)
+
+// coinpaymentsTransactions is Coinpayments transactions DB.
+type coinpaymentsTransactions struct {
+	db *dbx.DB
+}
+
+// Insert inserts new coinpayments transaction into DB.
+func (db *coinpaymentsTransactions) Insert(ctx context.Context, tx stripecoinpayments.Transaction) (*stripecoinpayments.Transaction, error) {
+	amount, err := tx.Amount.GobEncode()
+	if err != nil {
+		return nil, err
```

Comment:
Shouldn't we have to wrap all the errors returned by the methods of this type?

Reply:
Don't we have a wrapper for all `dbx` errors? If that is the policy we follow that I will add another error

Reply:
Checking `dbx` autogenerated code could be because there is an option to set a wrap error function. I don't know if it's set, I suppose that it's

Anyway, this one and some others of this function aren't errors returned by DBX so they should be wrapped.